webapi/src/main/scala/org/knora/webapi/responders/v1/UsersResponderV1.scala

```diff
@@ -531,33 +502,33 @@ class UsersResponderV1(responderData: ResponderData) extends Responder(responder
 
       for {
         /* get the user's permission profile from the permissions responder */
-        permissionData <- appActor
-                            .ask(
-                              PermissionDataGetADM(
-                                projectIris = projectIris,
-                                groupIris = groupIris,
-                                isInProjectAdminGroups = isInProjectAdminGroups,
-                                isInSystemAdminGroup = isInSystemAdminGroup,
-                                requestingUser = KnoraSystemInstances.Users.SystemUser
-                              )
-                            )
-                            .mapTo[PermissionsDataADM]
-
-        maybeProjectInfoFutures: Seq[Future[Option[ProjectInfoV1]]] =
+        permissionData <-
+          messageRelay
+            .ask[PermissionDataGetADM](
+              PermissionDataGetADM(
+                projectIris = projectIris,
+                groupIris = groupIris,
+                isInProjectAdminGroups = isInProjectAdminGroups,
+                isInSystemAdminGroup = isInSystemAdminGroup,
+                requestingUser = KnoraSystemInstances.Users.SystemUser
+              )
+            )
+            .as(PermissionsDataADM())
```

Comment:
This line is causing your some test to fail. You should remove it.

Basically this line tells to ignore response the relay and replaces both success and failure by an empty `PermissionDataADM()`.